pom.xml

```diff
@@ -25,7 +25,7 @@
 
     <groupId>com.connexta.ion.ingest</groupId>
     <artifactId>ingest-api</artifactId>
-    <version>0.2.1-SNAPSHOT</version>
+    <version>0.3.0-SNAPSHOT</version>
```

Comment:
FYI. The Jenkins release job can change version numbers as part of the release.

Reply:
Yes. We agreed that we should either bump the version as part of the PR or release immediately after the PR is merged. In this case, I chose the first option because we don't need the new release right now.